sql/src/main/java/io/crate/analyze/InsertFromValuesAnalyzer.java

```diff
@@ -271,7 +271,7 @@ private void analyzeValues(DocTableRelation tableRelation,
                                ValuesResolver valuesResolver,
                                ExpressionAnalyzer valuesAwareExpressionAnalyzer,
                                ValuesList node,
-                               List<Assignment> assignments,
+                               @javax.annotation.Nullable List<Assignment> onDuplicateKeyAssignments,
```

Comment:
replace `org.elasticsearch.common.Nullable` import with `javax.annotation.Nullable`

Reply:
Would replace the other Nullables with javax.annotation.Nullable. AFAIK it is the standard in the Java world.

Reply:
Even better. Sorry, misread that.